src/components/AddBook.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const AddBook = () => {
+  const categories = [];
+  return (
+    <>
+      <h1 className="header">Add new book </h1>
+      <form>
+        <input type="text" className="bookTitle" placeholder="Book Title" />
+        <select name="categorie" id="categorie">
+          {categories.map((category) => (
+            <option key={category.id} value={category.name}>{category.name}</option>
+          ))}
+        </select>
+        <button type="submit" className="submitBtn">Add Book</button>
+      </form>
```

Comment:
- [ ] For the component to add new books, it renders a form with inputs for the title and the author of the book. You are missing the author input field.